src/storybook/functions/createStoryMetaSettings.js

```diff
@@ -14,6 +14,14 @@ const allowedIcons = iconsMetaData.reduce(
   { options: [], mapping: {} }
 );
 
+function cleanTSEnums(enumObj) {
```

Comment:
Until now all enums had string values because backward compatibility, this is the first enum which does not have this issue. Do we still want to define string values for each ts enum key? or we are prefer to use number values?
If we prefer using number values we need to remove them for the options object because TS enums with number values contains both keys and values as keys.

Reply:
If we prefer to continue using string values I will revert this change